### ISSUE TITLE
dataio-pyshovel: `?` is search up in my configuration, not help (Trac #1638)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1630]
Like the vim keybindings, `?` is search above cursor, not help as suggested in the top right. 

Migrated from https://code.icecube.wisc.edu/ticket/1638
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:51:53", 
    "description": "Like the vim keybindings, `?` is search above cursor, not help as suggested in the top right. ", 
    "reporter": "rmaunu", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547243513023870", 
    "component": "combo core", 
    "summary": "dataio-pyshovel: `?` is search up in my configuration, not help", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-04-17T14:00:44", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160417T14:07:21: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed owner from "" to "david.schultz"
* david.schultz commented:

> In changeset 2517:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2517"
> fix help text to 'h'. closes #1630
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:51:53: nwhitehorn commented:

> In changeset 169190:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169190"
> fix help text to 'h'. closes #1630
> ```
